### ISSUE TITLE
Used Links Color

### ISSUE THREAD

**Issue by FableDNK** [liorgrossman/darkness#184]
I suggest adding a slightly different color to links that one has already clicked on, same as by default.

**Comment by theme1256** [comment]
This is already implemented as part of all themes.  
Some creators of designs might have chosen to not use it.

From `material.scss`:
```scss
// Links color
$theme-link: invert-back-color(#64B5F6);
$theme-link-visited: invert-back-color(#7986CB);
$theme-link-active: invert-back-color(#90CAF9);
```

**Comment by liorgrossman** [comment]
Agreed, @FableDNK - can you please elaborate? (what site/page are we talking about, what theme?)

**Comment by FableDNK** [comment]
I just mean Google search. I've attached some images.
